### PR TITLE
Pop shop documentation

### PR DESCRIPTION
**Pull request by realKarthikNair**

### CODE REVIEW

**Comment by ahoneybun** [comment]
It looks like you have the Krita change in this PR so it would need to not be in this PR as well: https://github.com/system76/docs/pull/1102

File: content/pop-shop.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: pop
+tableOfContents: true
+---
+
+Welcome to Pop!\_Shop - the software center for Pop!\_OS. Pop!\_Shop is a user-friendly graphical interface for installing and removing software on your computer. Pop!\_Shop is based on Elementary AppCenter
```
Comment (jacobgkau) [inline content/pop-shop.md:23]: The last sentence here is missing punctuation.

File: content/pop-shop.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+![Pop!_Shop](/images/pop-shop/pop-shop.png)
+
+## Searching for Software
+
+To search for software, type the name of the software you are looking for in the search bar at the top of the window.
+
+![Pop!_Shop](/images/pop-shop/pop-shop-search.png)
+
+## Installing Software
```
Comment (jacobgkau) [inline content/pop-shop.md:37]: How much value is there in listing the buttons of the application in an article? These sentences aren't really adding anything beyond the name of the button.

File: content/pop-shop.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+![Pop!_Shop](/images/pop-shop/pop-shop-uninstall-software.png)
+
+## Updating installed software
+
+To update installed software, go to the `Installed` tab and click the `Update` button next to the software you want to update.
+
+![Pop!_Shop](/images/pop-shop/pop-shop-update-software.png)
+
+## Managing Repositories
```
Comment (jacobgkau) [inline content/pop-shop.md:55]: We have a separate article for Repoman already: https://support.system76.com/articles/manage-repos-pop

**Review by jacobgkau — CHANGES_REQUESTED** [review]

**Pull request closed without merging**